.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2026-03-22 - Werkzeug DoS Vulnerability (CVE-2026-27199)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)